resources/views/planner/planner.blade.php

```diff
@@ -63,6 +63,9 @@
             <!-- Card body -->
             <div class="card-body">
                 <form>
+
+                    <!-- // TODO search core_team (≈ bookmarking) -->
```

Comment:
I guess Till would suggest here, to put that in you Kanban-Board... But who gets by without the good old post-its everywhere? 😉